### PR TITLE
refactor: manage font weight & font family in @twreporter/core

### PR DESCRIPTION
**Pull request by Aylie-Chou**
- add @twreporter/core/lib/constants/font
- deprecate @twreporter/core/lib/constants/font-weight

### CODE REVIEW

**Review by taylrj — APPROVED** [review]
LGTM

File: packages/core/src/constants/font.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  normal: '400',
+  bold: '700',
+}
+
+const fontFamily = {
+  title: 'merriweather, source-han-serif-tc, serif',
+  default: 'rosario, source-han-sans-traditional, sans-serif',
+}
+
+export default { fontWeight, fontFamily }
```
Comment (taylrj) [inline packages/core/src/constants/font.js:12]: Minor. Do we need default export if our use cases seem to use named import ?

Comment (Aylie-Chou) [reply packages/core/src/constants/font.js:12]: Default export is not neccessary in current use cases.
Do you think we would need this kind of flexibility ?

Comment (taylrj) [reply packages/core/src/constants/font.js:12]: Yes, we could keep this flexibility for future use.

**Pull request merged**